app/api/custom/orders.py

```diff
@@ -87,7 +87,12 @@ def calculate_order_amount_wrapper(data):
     if 'discount-code' in data:
         discount_code_id = data['discount-code']
         discount_code = safe_query(db, DiscountCode, 'id', discount_code_id, 'id')
+<<<<<<< HEAD
+        if not TicketingManager.match_discount_quantity(discount_code, tickets, None):
+            raise UnprocessableEntityError({'source': 'discount-code'}, 'Discount Usage Exceeded').respond()
+=======
     return tickets, discount_code
+>>>>>>> 3ea38141a731cf51a3c8314e02b5ec38d0b8e932
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Possible SCM diff in code](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4712256)